core/src/test/java/org/elasticsearch/index/translog/TranslogTests.java

```diff
@@ -1008,42 +1032,6 @@ public void testTranslogWriter() throws IOException {
         IOUtils.close(writer);
     }
 
-    public void testFailWriterWhileClosing() throws IOException {
```

Comment:
yeah, I get why you removed it but It think it's indeed good to keep around - it seems we don't have any other test for the closeIntoReader method.